packages/forms/frontend/sirius-components-widget-reference/src/ReferencePropertySection.tsx

```diff
@@ -34,14 +38,22 @@ import {
   GQLEditReferenceVariables,
   GQLErrorPayload,
   GQLReferenceWidget,
+  GQLReferenceWidgetStyle,
   GQLSuccessPayload,
 } from './ReferenceWidgetFragment.types';
 
-const useStyles = makeStyles((_) => ({
+const useStyles = makeStyles<Theme, GQLReferenceWidgetStyle>(() => ({
   root: {
     overflow: 'auto',
     maxHeight: 300,
   },
+  style: {
+    color: ({ color }) => (color ? color : 'inherit'),
```

Comment:
All those `'inherit'` will create issues since it will not fallback to the original style but to the style of the parent which may not be related at all. @AxelRICHARD is fixing exactly that in another pull request